redis_rate_limit/__init__.py

```diff
@@ -88,7 +92,6 @@ def get_usage(self):
         """
         Returns actual resource usage by client. Note that it could be greater
         than the maximum number of requests set.
-
```

Comment:
Blank line is still removed, could you revert it?

Reply:
Done

Reply:
Have you pushed those updates?

Reply:
Sorry forgot to push changes. Updated.